packages/services/src/services/Analytics/Analytics.js

```diff
@@ -213,7 +213,7 @@ class AnalyticsAPI {
    *       currentTime: 1,
    *       duration: 60,
    *       playerState: 1,
-   *       videoId: '1_9h94wo6b',
+   *       mediaId: '1_9h94wo6b',
```

Comment:
Is this technically an API change? Would probably need to note this in `breaking-changes.md` in the `services` package. (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/services/breaking-changes.md).

cc: @annawen1